server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/features/ResetFeatureStateResponse.java

```diff
@@ -43,7 +44,7 @@ public ResetFeatureStateResponse(StreamInput in) throws IOException {
         this.resetFeatureStateStatusList = in.readList(ResetFeatureStateStatus::new);
     }
 
-    public List<ResetFeatureStateStatus> getItemList() {
+    public List<ResetFeatureStateStatus> getFeatureStateResetStatusList() {
```

Comment:
```suggestion
    public List<ResetFeatureStateStatus> getFeatureStateResetStatuses() {
```

Reply:
Thanks for catching this. I may read some _Effective Java_ as penance for this mistake.